autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionInfo.tsx

```diff
@@ -120,11 +120,24 @@ const SectionInfo: React.FC<SectionInfoProps> = ({
     </Typography>
   );
 
+  // adds a tooltip for meeting types that aren't very obvious ex -> INS, PRL, etc.
+  const formatMeetingType = (mtg: Meeting): JSX.Element => {
+    const meetingTypeDescription = MeetingTypeDescription.get(mtg.meetingType);
+    if (meetingTypeDescription != null) {
```

Comment:
```suggestion
    if (meetingTypeDescription) {
```
because it's shorter